NUAttribute.js

```diff
@@ -72,12 +72,12 @@ export default class NUAttribute extends NUObject {
         'this' would correspond to the NUEntity object being validated.
         NUAttribute object on which the validation needs to be executed is passed as args[0].
     */
-    validate(entity, attrObj, formValues) {
+    validate(entity, attrObj, formValues, operation) {
         
         if (attrObj) {
             const attrValue =  entity && entity[attrObj.name];
             //if STRING use !attrValue to check if value provided. For all other attribute types use !undefined and !null
-            if (attrObj.isRequired && !attrObj.isValueSet(attrValue)) {
+            if (attrObj.isRequired && !attrObj.isValueSet(attrValue) && !(attrObj.isPassword && operation === 'edit')) {
```

Comment:
@killanch In certain cases we want to have the password validation even on edit so this will make it impossible. What we should add is the possibility of skipping validation on any field.

Reply:
@natabal Im now passing optionalAttrs array which will be used to skip mandatory attrs validation, can you pls review the PRs?